core/src/main/scala/akka/persistence/cassandra/journal/TagWriters.scala

```diff
@@ -312,18 +326,7 @@ import akka.util.ByteString
           }
         }
 
-        result.onComplete {
-          case Success(_) =>
-            scheduleWriteTagScanningTick()
-            log.debug(
-              "Update tag scanning of [{}] pids took [{}] ms",
-              updates.size,
-              (System.nanoTime() - startTime) / 1000 / 1000)
-          case Failure(t) =>
-            scheduleWriteTagScanningTick()
-            log.warning("Writing tag scanning failed. Reason {}", t)
-            self ! TagWriteFailed(t)
-        }
+        result.onComplete(self ! WriteTagScanningCompleted(_, startTime, updates.size))
```

Comment:
:+1:

Reply:
I missed something here. It should also `self ! TagWriteFailed(t)`. I'll update.